docs/rst/amldashboard/specification/amldashboard.rst

```diff
@@ -0,0 +1,30 @@
+.. include:: /rst/exports/alias.include
+.. include:: /rst/exports/roles.include
+
+.. _amldashboard_specification:
+
+#############
+AML Dashboard
+#############
+
+The *AML Dashboard* serves as a user-friendly tool for training, evaluating, and applying |aml| models to expressive gestures in music exploration.
+Built upon the `Marcelle <https://marcelle.dev/>`__ framework, the dashboard offers standard interaction :term:`GUI`\s and the ability to create custom interactive widgets compatible with Marcelle's structure.
+
+The primary goal of the *AML Dashboard* is to enable users to:
+
+1. Collect personal gesture data.
+2. Train classifiers using the *AML Engine*.
+3. Apply classified gesture data in a music exploration context.
+
+To achieve this, the *AML Dashboard* integrates four main interfaces:
+
```

Comment:
Please add a reference to each tab in the dashboard (figures, or real name of the tab as the way it is really found in the Dashboard)